### PR TITLE
Modifications to program certificate

### LINKED ISSUE #4852: More updates to MicroMasters Program Certificate
Our Program Certificate still doesn't match the edX program certificate. 

#### Designs and Mockups

links coming

#### Acceptance Criteria:

- [ ] "has successfully completed all courses and received passing grades to earn a MicroMasters program certificate in" 
- [ ] Underneath the certificate, the edX version explains more details about what a MicroMasters is:
"ABOUT THE EDX MICROMASTERS CREDENTIAL An edX MicroMasters credential signifies that the learner has completed a group of courses that add up to a rich understanding of an area of study. Completion of MicroMasters Programs requires that learners earn Verified Certificates from each of the courses included in the Program."

### PR DESCRIPTION
**Pull request by annagav**
- [ ] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review

#### What are the relevant tickets?
Fix #4852

#### What's this PR do?
Modified the text of the certificate and added more information to the footer.

#### How should this be manually tested?
Create a program certificate for your user, and in the cms create program certificate signatories for the user. Then go to the user dashboard to view the certificate.

Screen Shot 2021-04-01 at 4 18 11 PM
Screen Shot 2021-04-01 at 4 15 36 PM
Screen Shot 2021-04-01 at 4 23 50 PM
Screen Shot 2021-04-01 at 4 24 06 PM

### CODE REVIEW

File: ui/templates/certificate.html [hunk 1]
```diff
@@ -147,5 +148,30 @@ <h4 class="signatory-name">{{ signatory.name }}</h4>
...
+        </h3>
+        <p class="accomplishment-metadata-copy">
+          An edX MicroMasters credential signifies that the learner has completed a group of courses that add up
+          to a rich understanding of an area of study. Completion of MicroMasters Programs requires that learners earn
+          <a href="https://www.edx.org/verified-certificate">Verified Certificates</a>
+          from each of the courses included in the Program.
+      </p>
+    </div>
+  </div>
+</div>
```
Comment (pdpinch) [inline ui/templates/certificate.html:175]: This isn't included in the printed version at edX, so I'd like to make sure that we don't include it in the printed version either.

File: ui/templates/certificate.html [hunk 2]
```diff
@@ -147,5 +148,30 @@ <h4 class="signatory-name">{{ signatory.name }}</h4>
...
+      </div>
+      <div class="accomplishment-metadata-bit accomplishment-metadata-aboutcert">
+        <h3 class="accomplishment-metadata-title">
+          About the edX MicroMasters credential
+        </h3>
+        <p class="accomplishment-metadata-copy">
+          An edX MicroMasters credential signifies that the learner has completed a group of courses that add up
+          to a rich understanding of an area of study. Completion of MicroMasters Programs requires that learners earn
+          <a href="https://www.edx.org/verified-certificate">Verified Certificates</a>
+          from each of the courses included in the Program.
```
Comment (pdpinch) [inline ui/templates/certificate.html:171]: At edx, it looks like this text is conditional on whether it's a course or a program certificate. 

On the course certificate it says: 
>
> About edX Verified Certificates
> 
> An edX Verified Certificate signifies that the learner has agreed to abide by the edX honor code and  completed all of the required tasks of this course under its guidelines, as well as having their photo ID checked to verify their identity.

**Comment by annagav** [comment]
Screenshot for the course certificate:
Screen Shot 2021-04-01 at 5 27 46 PM

**Review by HamzaIbnFarooq — APPROVED** [review]
LG. 
I don't know if it bothers anyone but mentioning it here anyway: the `i` in `certificate` might need some space:
image

**Pull request merged**